tracer/src/Datadog.Tracer.Native/dd_profiler_constants.h

```diff
@@ -101,6 +101,9 @@ const shared::WSTRING skip_traceattribute_assembly_prefixes[]{
 const shared::WSTRING skip_assemblies[]{WStr("mscorlib"),
                                 WStr("netstandard"),
                                 WStr("System.Configuration"),
+                                WStr("System.Security.AccessControl"),
+                                WStr("System.Security.Claims"),
+                                WStr("System.Security.Principal.Windows"),
```

Comment:
Hmmm... I don't _think_ this is doing what you think it is 🤔  As I understand it, this adds the assemblies to the "never instrument" list, but what we really want is to add them to the "don't add the startup hook" list. _Maybe_ that's actually what we want to do anyway, just as long as we don't _need_ to instrument `System.Security.Claims` for ASM (for example)?

I believe this is the code you were actually looking for (similar to what you did in https://github.com/DataDog/dd-trace-dotnet/pull/5910?) 

https://github.com/DataDog/dd-trace-dotnet/blob/eb4d6ec945843be865eec9f9e495efd58f7eea4b/tracer/src/Datadog.Tracer.Native/cor_profiler.cpp#L1625-L1650

Reply:
Good call, let me test that approach. That will definitely have a more targeted impact

Reply:
Yep that also works and is more targeted. Updated in 2848bc38ba69681d655f68546637ba61ba98eaa1